scripts/variantstore/wdl/GvsValidateVAT.wdl

```diff
@@ -85,6 +93,68 @@ task EnsureVatTableHasVariants {
     }
 }
 
+task SpotCheckForExpectedTranscripts {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String? service_account_json_path
+        String last_modified_timestamp
+    }
+
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+
+    command <<<
+        if [ ~{has_service_account_file} = 'true' ]; then
+            gsutil cp ~{service_account_json_path} local.service_account.json
+            gcloud auth activate-service-account --key-file=local.service_account.json
+            gcloud config set project ~{query_project_id}
+        fi
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT
+            contig,
+            position,
+            vid,
+            gene_symbol,
+            variant_consequence
+        FROM
+            ~{fq_vat_table},
+            UNNEST(consequence) AS variant_consequence
+        WHERE
+            contig = "chr19" AND
+            position >= 35740407 AND
+            position <= 35740469 AND
+            variant_consequence NOT IN ("downstream_gene_variant","upstream_gene_variant") AND
+            gene_symbol NOT IN ("IGFLR1","AD000671.2")' > bq_query_output.csv
+
+        # get number of lines in bq query output
+        NUMRESULTS=$(awk 'END{print NR}' bq_query_output.csv)
+
+        # if the result of the query has any rows, that means there were unexpected transcripts at the
+        # specified location, so report those back in the output
+        if [[ $NUMRESULTS = "0" ]]; then
+            echo "PASS: The VAT table ~{fq_vat_table} only has the expected transcripts at the tested location ("IGFLR1" and "AD000671.2" in chromosome 19, between positions 35,740,407 - 35,740,469)." > validation_results.txt
+        else
```

Comment:
Should some of these quotation marks be escaped?

Reply:
It still seems to work, but I'm not sure why — changed the nested double quotes to single quotes within double quotes.